darts/tests/datasets/test_dataset_loaders.py

```diff
@@ -97,6 +97,15 @@
         pre_process_zipped_csv_fn=None,
     )
 )
+ele_multi_series_dataset = DatasetLoaderCSV(
+    metadata=DatasetLoaderMetadata(
+        "Electricity_test.csv",
+        uri=_DEFAULT_PATH + "/Electricity_test.csv",
```

Comment:
Could we maybe put this one in `datasets/test/Electricity_test.csv` ? i.e., we create a test subfolder.